modules/oe_content_event/src/Form/EventVenueForm.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_event\Form;
+
+use Drupal\Component\Datetime\TimeInterface;
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Entity\EntityRepositoryInterface;
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form controller for Event venue edit forms.
+ *
+ * @ingroup oe_content_event
+ */
+class EventVenueForm extends ContentEntityForm {
+
+  /**
+   * The current user account.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $account;
+
+  /**
+   * The messenger.
+   *
+   * @var \Drupal\Core\Messenger\MessengerInterface
+   */
+  protected $messenger;
```

Comment:
The messenger trait already has this protected variable declared. And the parent FormBase uses that so no need to redeclare it here.